packages/test-cases/cases/apis/wasm-rs/json-type/query/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+pub mod w3;
+use polywrap_wasm_rs::JSON;
+pub use w3::*;
+use std::string::String;
+
+pub fn parse(input: InputParse) -> JSON::Value {
+    JSON::from_str(&input.value).unwrap()
+}
+
+pub fn stringify(input: InputStringify) -> String {
+    let result_vec: Vec<String> = input.values.iter()
+    .map(|value| JSON::to_string(&value).unwrap())
+    .collect();
+
+    polywrap_wasm_rs::w3_debug_log(&format!("{:?}", result_vec));
```

Comment:
Should we add this to the "use" statement along with JSON?
Then we can just do `w3_debug_log(...);`